src/Configuration/SshParameters.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorCommon\Configuration;
+
+class SshParameters
+{
+    /** @var bool */
+    private $enabled;
+
+    /** @var array|null */
+    private $keys;
+
+    /** @var string|null */
+    private $sshHost;
+
+    /** @var int */
+    private $sshPort;
+
+    /** @var string|null */
+    private $remoteHost;
+
+    /** @var int|null */
+    private $remotePort;
+
+    /** @var int  */
+    private $localPort;
+
+    /** @var string|null */
+    private $user;
+
+    /** @var string|null */
+    private $privateKey;
+
+    public function __construct(
```

Comment:
Když už se to předělává, tak nestálo by za kontrolu to, jestli to všechno dává takhle smysl? Já vůbec nechápu k čemu bude nastavení SSH tunelu, když tomu nedám ten remote host/port

Reply:
Opet 
https://github.com/keboola/db-extractor-common/pull/94/files#r226447144

Respektuju stav jak to bylo, kvuli mozny BC breakum, zalozim issue primo na SSH tunel.

Reply:
@Actimel  souhlas, tohle nemuzeme libovolne menit, pokud dneska existuje konfigurace ve ktery to takhle je a funguje, tak bysme ji 'bezduvodne' rozbili